ecs/src/nox/ecs/component/Transform.h

```diff
@@ -0,0 +1,84 @@
+#ifndef NOX_ECS_TRANSFORM_H_
+#define NOX_ECS_TRANSFORM_H_
+
+#include <string>
+
+#include <nox/ecs/Component.h>
+#include <nox/ecs/EntityManager.h>
```

Comment:
No need to include the EntityManager in the header. It is not needed for the interface.